classes/s3_file_system.php

```diff
@@ -40,4 +40,11 @@ protected function get_external_client($config) {
         $s3client = new s3_client($config);
         return $s3client;
     }
+
+    public function copy_object_from_local_to_external_path($contenthash) {
+        $config = get_objectfs_config();
+        $s3client = $this->get_external_client($config);
+        $localpath = $this->get_local_path_from_hash($contenthash);
+        $s3client->copy_object_from_local_to_external_path($localpath, $contenthash);
```

Comment:
should 

return $s3client->copy_object_from_local_to_external_path($localpath, $contenthash);